app-crypt/moolticute/moolticute-0.8.0_beta.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/raoulh/moolticute.git"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/raoulh/moolticute/archive/v${PV//_/-}.tar.gz -> ${P}.tar.gz"
```

Comment:
I'd suggest keeping the archive name consistent with the directory inside it, i.e. `${P/_/-}`.

As a minor nitpick, you're using `//` for PV but a single `/` for P.